Jenkinsfile

```diff
@@ -0,0 +1,21 @@
+
+node {
+	def VERSION = sh(returnStdout: true, script: "git rev-parse HEAD").trim()
+	def CACHE_DIR = sh(returnStdout: true, script: "mktemp go-ipfs.cache.XXXXXXXX").trim()
```

Comment:
It is build cache that contains binary files that are shortcuts for building. It could be a layer if we forced a build in the container creation, but then build failure would be quite cryptic.